src/observers/strategies/seniority.cpp

```diff
@@ -1,29 +1,87 @@
 #include <list>
 #include <utility>
 
+#include "bus/event_bus.hpp"
+
 #include "observers/strategies/seniority.hpp"
 
+#include "serenity/resource_helper.hpp"
+
 namespace mesos {
 namespace serenity {
 
 using std::list;
 using std::pair;
 
+inline void setOpenEstimatorPipeline(bool opened) {
```

Comment:
This function can be helpful for everyone, and let's make it available, before everybody will do copy-paste of this code.
It could be ```static Try<Nothing> StaticEventBus::PublishOversubscriptionCtrlEvent(bool);```

Also, I'm not sure about the name ```Event```. Shouldn't it rather be message?